pkgs/development/libraries/qscintilla-qt4/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, lib, fetchurl, unzip, qt4, qmake4Hook
+}:
+
+let
+  pname = "qscintilla-qt4";
+  version = "2.11.6";
```

Comment:
Put this few lines below instead of `inherit pname version;` and you can drop the whole `let` block.